packages/create-expo-nightly/src/Packages.ts

```diff
@@ -77,7 +77,7 @@ export async function getExpoPackagesAsync(expoRepoPath: string): Promise<Packag
   }
 
   const paths = await Array.fromAsync(
-    fs.promises.glob('**/package.json', {
+    fs.promises.glob(['*/package.json', '@expo/*/package.json'], {
       cwd: path.join(expoRepoPath, 'packages'),
```

Comment:
not a fan to limit the covention
maybe just add ` '@expo/cli/local-template/**',` to the exclude below